### PR TITLE
Add recommendations in system requirements

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
Recommend using PHP 7.x & setting `memory_limit=256M`

### CODE REVIEW

File: src/content/1.7/basics/installation/system-requirements.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ weight: 10
...
-* MySQL: 5.0 or later.
-* Server RAM: The more the better…
+* **System:** Unix, Linux or Windows.
+* **Web server:** Apache Web Server 2.2 or any later version.
+* **PHP:**
+    - {{% minver v="1.7.0~1.7.3" %}} PHP 5.4 minimum, PHP 7.0 recommended
+    - {{% minver v="1.7.4" %}} PHP 5.6 minimum, PHP 7.1 recommended
+    - {{% minver v="1.7.5+" %}} PHP 5.6 minimum, PHP 7.2 recommended
+* **MySQL:** 5.0 or later.
+* **Server RAM:** The more the better. We recommend setting the memory allocation per script (`memory_limit`) to `256M`.
```
Comment (matks) [inline src/content/1.7/basics/installation/system-requirements.md]:
```suggestion
* **Server RAM:** The more the better. We recommend setting the memory allocation per script (`memory_limit`) to minimum `256M`.
```

File: src/content/1.7/basics/installation/system-requirements.md [hunk 2]
```diff
@@ -8,13 +8,14 @@ weight: 10
...
-    - {{% minver v="1.7.4" %}} PHP 5.6 or later
-* MySQL: 5.0 or later.
-* Server RAM: The more the better…
+* **System:** Unix, Linux or Windows.
+* **Web server:** Apache Web Server 2.2 or any later version.
+* **PHP:**
+    - {{% minver v="1.7.0~1.7.3" %}} PHP 5.4 minimum, PHP 7.0 recommended
+    - {{% minver v="1.7.4" %}} PHP 5.6 minimum, PHP 7.1 recommended
+    - {{% minver v="1.7.5+" %}} PHP 5.6 minimum, PHP 7.2 recommended
+* **MySQL:** 5.0 or later.
```
Comment (PierreRambaud) [inline src/content/1.7/basics/installation/system-requirements.md]: Should be 5.6+ instead, (was release in February 2013, whereas 5.0 was release in 2005 🙄 )

Comment (matks) [reply src/content/1.7/basics/installation/system-requirements.md]: You dont like vintage, do you ?

Comment (PierreRambaud) [reply src/content/1.7/basics/installation/system-requirements.md]: If you knew ;)

**Comment by mickaelandrieu** [comment]
LGTM: thanks everyone!

**Pull request merged**